semgrep-core/src/osemgrep/cli_scan/Cli_json_output.mli

```diff
@@ -0,0 +1,7 @@
+val cli_output_of_core_results :
+  Scan_CLI.conf -> Core_runner.result -> Semgrep_output_v0_j.cli_output
+
+val lines_of_file :
```

Comment:
maybe some comments about lines_of_file, or use Common.filename instead of string.

Reply:
https://github.com/returntocorp/semgrep/pull/6399